site/docs/contribute.md

```diff
@@ -150,8 +150,6 @@ Example:
 
 ### Style
 
-For Python, please use the tox command `tox -e format` to apply autoformatting to the project.
```

Comment:
This doc came from `Iceberg-docs` repo. Which is a common website for all the Iceberg repo. So, I think we don't need to remove this.

Reply:
PyIceberg moved away from `tox` a long time ago, so I think we're good to get rid of this. We've replaced this by the [pre-commit](https://py.iceberg.apache.org/contributing/#linting) framework. Also, this should live in the https://github.com/apache/iceberg-python repository. Let me know if you think differently.

Reply:
> PyIceberg moved away from tox a long time ago,

I didn't know this. LGTM then.